main.go

```diff
@@ -43,6 +43,8 @@ func setupRouter(sqlDb *sqlx.DB) *gin.Engine {
 
 	router.POST("/api/send-career-mail", jobsRepo.SendCareerMail)
 
+	router.GET("/sitemap", jobsRepo.GenerateSitemap)
```

Comment:
I think sitemap generation is not responsibility of the jobs repo. We should create new module for it.